[2025-01-28T11:26:09.277+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T11:26:09.291+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [queued]>
[2025-01-28T11:26:09.297+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [queued]>
[2025-01-28T11:26:09.298+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2025-01-28T11:26:09.308+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): ingest_data> on 2025-01-28 11:22:10.902666+00:00
[2025-01-28T11:26:09.315+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_data_ingestion', 'ingest_data', 'manual__2025-01-28T11:22:10.902666+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_ingestion.py', '--cfg-path', '/tmp/tmpc_d1j3yj']
[2025-01-28T11:26:09.317+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask ingest_data
[2025-01-28T11:26:09.318+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=346) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T11:26:09.319+0000] {standard_task_runner.py:72} INFO - Started process 347 to run task
[2025-01-28T11:26:09.353+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T11:22:10.902666+00:00 [running]> on host 0711bf72f9ce
[2025-01-28T11:26:09.440+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T11:22:10.902666+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T11:22:10.902666+00:00'
[2025-01-28T11:26:09.441+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T11:26:09.452+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T11:26:09.452+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'ls /opt/***/data/']
[2025-01-28T11:26:09.462+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T11:26:09.466+0000] {subprocess.py:106} INFO - yellow_2022_01_data.parquet
[2025-01-28T11:26:09.466+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T11:26:09.490+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T11:26:09.490+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_data_ingestion, task_id=ingest_data, run_id=manual__2025-01-28T11:22:10.902666+00:00, execution_date=20250128T112210, start_date=20250128T112609, end_date=20250128T112609
[2025-01-28T11:26:09.532+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T11:26:09.547+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T11:26:09.548+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
